recipes/base_install.rb

```diff
@@ -217,15 +225,19 @@
 # Install NVIDIA and CUDA
 include_recipe "aws-parallelcluster::_nvidia_install"
 
-# Install FSx options
-include_recipe "aws-parallelcluster::_lustre_install"
+# FIXME:
+# # Install FSx options
+# include_recipe "aws-parallelcluster::_lustre_install"
```

Comment:
You can re-enable it, I merged the https://github.com/aws/aws-parallelcluster-cookbook/pull/753

Reply:
Enabled.